assignment-1/Team.java

```diff
@@ -0,0 +1,27 @@
+import java.util.ArrayList;
+
+public class Team extends Players {
```

Comment:
1. Why does team extend player? 
2. did not see implementation  of hashcode and equals method.